ServerCore/Pages/Teams/Play.cshtml

```diff
@@ -0,0 +1,73 @@
+﻿@page "/{eventId}/Teams/Play"
+@model ServerCore.Pages.Teams.PlayModel
+
+@{
+    ViewData["Title"] = "Play";
+}
+
+<h2>Play</h2>
+
+<table class="table">
+    <thead>
+        <tr>
+            <th>
+                <a asp-page="./Play" asp-route-id="@Model.TeamID" asp-route-sort="@(Model.SortForColumnLink("puzzle"))">
+                    @Html.DisplayNameFor(model => model.PuzzlesWithState[0].Puzzle.Name)
+                </a>
+            </th>
+            <th>
+                <a asp-page="./Play" asp-route-id="@Model.TeamID" asp-route-sort="@(Model.SortForColumnLink("group"))">
+                    @Html.DisplayNameFor(model => model.PuzzlesWithState[0].Puzzle.Group)
+                </a>
+            </th>
+            <th>
+                @Html.DisplayNameFor(model => model.PuzzlesWithState[0].Puzzle.OrderInGroup)
+            </th>
+            <th>
+                @Html.DisplayNameFor(model => model.PuzzlesWithState[0].Puzzle.PuzzleUrl)
+            </th>
+            <th>
+                @Html.DisplayNameFor(model => model.PuzzlesWithState[0].Puzzle.AnswerUrl)
+            </th>
+            <th>
+                <a asp-page="./Play" asp-route-id="@Model.TeamID" asp-route-sort="@(Model.SortForColumnLink("solve"))">
+                    Solve
+                </a>
+            </th>
+        </tr>
+    </thead>
+    <tbody>
+@foreach (var item in Model.PuzzlesWithState) {
+        <tr>
+            <td>
+                @Html.DisplayFor(modelItem => item.Puzzle.Name)
+            </td>
+            <td>
+                @Html.DisplayFor(modelItem => item.Puzzle.Group)
+            </td>
+            <td>
+                @Html.DisplayFor(modelItem => item.Puzzle.OrderInGroup)
+            </td>
+            <td>
+                @Html.DisplayFor(modelItem => item.Puzzle.PuzzleUrl)
+            </td>
+            <td>
+                @Html.DisplayFor(modelItem => item.Puzzle.AnswerUrl)
```

Comment:
Is this an admin page or a player page? If it's a player page, maybe this should be conditional on the event's answers available time

Reply:
It's a player page, hence the name "Play". I had a rationale at the time for skipping the conditional on answers, but I can't think of what that rationale could be at the moment. It was probably misguided at the time, will do.